src/lib/logger.ts

```diff
@@ -46,6 +46,7 @@ interface LogMessage {
   event: string;
   method: string;
   parameters: any;
+  activityUrl?: string;
```

Comment:
You haven't flagged me for re-review, but I would still recommend moving this up between the `application` and ``run_remote_endpoint` properties with which it's semantically related.